script/setup.js

```diff
@@ -63,9 +63,17 @@ try {
 
 	const { defaultOwner, defaultRepository } = await getDefaultSettings();
 
+	intro(
+		chalk.blue(
+			"Enter the details to hydrate your repository for your new package!"
```

Comment:
[Nit] Adding a bit more contextual info to this message... what do you think?

```suggestion
			"Let's collect some information to fill out repository details..."
```

Reply:
Changes done.